### DISCUSSION TITLE
Add custom Embed colours

### DISCUSSION THREAD

**Discussion by Vyvy-vi** [EddieHubCommunity/support#474] [First-Timers] [answered]
**Good first issue**- add custom Embeds
STATUS: issue opened
_ANSWERED BY_: @GoluffyxD 

DETAILS:
All current embeds made by EddieBot use Blue coloured Embed side-bars. Though these look good, it would be really great if we could get custom colour embeds(for eg: the `profile` command getting the colour of the user's toprole)

You can open an issue and maybe PR a fix on [EddieBot repo](https://github.com/EddieJaoudeCommunity/EddieBot) 🟩 
It would be great if we could pool ideas to customise the embeds and add a few fancy toppings to EddieBot.

## Screenshot of the current embeds:
![Screenshot 2020-12-03 at 3 57 47 AM](https://user-images.githubusercontent.com/62864373/100939137-cdea3700-351b-11eb-9290-b9b1526e3399.png)

## How to set embed-colour?
![Screenshot 2020-12-03 at 4 01 07 AM](https://user-images.githubusercontent.com/62864373/100939424-4224da80-351c-11eb-9637-c4bbffb0c7e9.png)

You can check out the [discord.js](https://discord.js.org/#/) docs for more info on [embeds](https://discordjs.guide/popular-topics/embeds.html#using-the-richembedmessageembed-constructor) at https://discordjs.guide/popular-topics/embeds.html#using-the-richembedmessageembed-constructor

For interesting colour choices check out: [Coolors](https://coolors.co/) 
## Additional Context:
![Screenshot 2020-12-03 at 3 50 53 AM](https://user-images.githubusercontent.com/62864373/100938788-4ef4fe80-351b-11eb-9a7e-c8dfe35d50ca.png)

**Comment by AllanRegush** [comment, marked as answer]
This feels like an issue. Should we create on Eddiebot?

**Reply by eddiejaoude** [reply]
I think this is a great idea 👍 

We could use red, amber, green.

For example
- standup: green
- ... : amber
- alexjs: red

Yep lets raise as an issue, it would be a `good first issue` too

**Comment by kiranmhm** [comment]
This issue still hasn't been raised. Should I raise the issue?

**Reply by AllanRegush** [reply]
Please do!

**Reply by kiranmhm** [reply]
Done.
Issue link: https://github.com/EddieJaoudeCommunity/EddieBot/issues/372

**Comment by Vyvy-vi** [comment]
(The intention of this was to provide `good first issue` for opening)
Forgot to write that 😅

**Reply by eddiejaoude** [reply]
We all got too excited to have colours in the bot's responses 😂

**Reply by mstanciu552** [reply]
Yeah it really sounded like an amazing feature 😂 😄 .
